libs/components/src/next/form/radio/stories/vl-radio.stories-doc.mdx

```diff
@@ -0,0 +1,45 @@
+import { ArgsTable, DocsStory, PRIMARY_STORY } from '@storybook/addon-docs';
+
+# Radio
+
+Gebruik het `radio` component om de gebruiker de mogelijkheid te geven om 1 of meerdere dingen te selecteren in een lijst.<br/>
```

Comment:
1. de component
2. Kunnen er meerdere dingen geselecteerd worden? Is een radio normaal gezien niet single-choice?